cypress/integration/seededFlows/articleFlows/hideArticleComments.spec.js

```diff
@@ -9,29 +9,34 @@ describe('Hiding/unhiding comments on an article', () => {
   });
 
   describe('Admin visits the article authored by them', () => {
-    it('Hides a comment and then unhides it from the same screen', () => {
+    it('Allows a user to hide/unhide a comment', () => {
       cy.findByRole('button', { name: 'Toggle dropdown menu' }).click();
       cy.findByRole('button', { name: "Hide Admin McAdmin's comment" }).click();
+
+      // Check we also expose the link to report abuse
+      cy.findByRole('link', { name: 'reporting abuse' })
+        .should('have.attr', 'href')
+        .and('contains', `/admin_mcadmin/comment/`);
+
       cy.findByRole('button', { name: 'Confirm' }).click();
 
-      cy.findByRole('button', { name: 'Toggle dropdown menu' }).should(
-        'not.be.visible',
-      );
+      // Page reloads after confirm click - we check the new page references the hidden comment
+      cy.findByText(
+        /Some comments have been hidden by the post's author/,
+      ).should('exist');
+
       cy.findByRole('img', { name: 'Expand' }).click();
       cy.findByRole('button', { name: 'Toggle dropdown menu' }).click();
       cy.findByRole('link', {
         name: "Unhide Admin McAdmin's comment",
       }).click();
-      cy.findByRole('img', { name: 'Expand' }).should('not.exist');
-    });
 
-    it('has report-abuse link  when showing hiding comment modal', () => {
```

Comment:
I incorporated the two specs together as it's quite a slow set of tests and we don't really need one just to check for presence of a link